src/ops/linalg_ops_test.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import * as tf from '../index';
+import { describeWithFlags } from '../jasmine_util';
+import { Tensor1D, Tensor2D } from '../tensor';
+import { ALL_ENVS, expectArraysClose, WEBGL_ENVS } from '../test_util';
+
+describeWithFlags('gramSchmidt-tiny', ALL_ENVS, () => {
+  // TODO(cais): Small things should get tested on ALL_ENVS.
+  it('2x2, Array of Tensor1D', () => {
+    const xs: Tensor1D[] = [tf.tensor1d([1, 2]), tf.tensor1d([-1, 5])];
```

Comment:
This might be a good candidate for a random array as input

Reply:
Done. Using random vectors and matrics in this test file, wherever it makes sense.